pkg/provider/aws/execution_unit.go

```diff
@@ -51,7 +51,17 @@ func (a *AWS) GenerateExecUnitResources(unit *core.ExecutionUnit, result *core.C
 		dag.AddDependency2(lambdaFunction, image)
 		return nil
 	case Kubernetes:
-		return nil
+		// transform kubernetes resources for EKS
+		for _, res := range dag.ListResources() {
+			if res.Provider() != "kubernetes" {
+				continue
+			}
+			for _, ref := range res.KlothoConstructRef() {
+				if ref.ToId() == unit.ToId() {
+					a.MapResourceDirectlyToConstruct(res, unit)
```

Comment:
this would put a reference on every helm chart for every exec unit? Im not sure thats what we want because we store which helm charts are correlated to the exec units

Reply:
it maps a kubernetes resource (I'll narrow this down to just charts in an update) directly to any execution unit whose `Id` matches any of the resource's `ConstructRefs` -- not every execution unit.